static/js/server_events.js

```diff
@@ -293,6 +293,9 @@ function dispatch_normal_event(event) {
             // than requiring a reload or page resize.
             page_params.default_language = event.setting;
         }
+        if (event.setting_name === 'timezone') {
+            page_params.timezone = event.setting;
```

Comment:
This commit's message mentions a "timzone", which sounds like a pretty awesome, but probably should be changed to a "timezone" :)